app/src/pages/prompt/promptCodeSnippets.tsx

```diff
@@ -233,9 +233,12 @@ export const mapPromptToSnippet = ({
   promptVersion: PromptVersion;
   language: CodeLanguage;
 }) => {
-  const generator = promptCodeSnippets[language][promptVersion.modelProvider];
+  const generator =
+    promptCodeSnippets[language][
+      promptVersion.modelProvider?.toLocaleLowerCase()
```

Comment:
the casing changed on me, not sure if gql is uppercasing them now but either way we can just lowercase them at all times in the ui